Gems/Compression/Code/Include/Compression/CompressionInterfaceAPI.h

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Interface/Interface.h>
+#include <AzCore/std/containers/span.h>
+#include <Compression/CompressionInterfaceStructs.h>
+
+namespace Compression
+{
+    enum class CompressionResult : u8
```

Comment:
Is this `AZ::u8`?  If so, how is the `AZ` namespace being found?
(Repeated)

Reply:
It is not, this code isn't being compiled yet, because nothing is including this header.
Will fix.